src/main/java/ru/naumen/sd40/log/parser/Parsers/GC/GCDataType.java

```diff
@@ -0,0 +1,17 @@
+package ru.naumen.sd40.log.parser.Parsers.GC;
```

Comment:
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

> Package names are written in all lower case to avoid conflict with the names of classes or interfaces